### PR TITLE
ZRZ-649: Add Custom MC Page

### PR DESCRIPTION
**Pull request by tomkln**
As per internal ZRZ-649.

### CODE REVIEW

File: content/zaraz/_index.md [hunk 1]
```diff
@@ -25,6 +25,10 @@ Offload third-party tools and services to the cloud and improve the speed and se
 You can add many third-party tools to Zaraz, and offload them from your website.
 {{</feature>}}
 
+{{<feature header="Custom Managed Components" href="/zaraz/get-started/add-custom-mc/">}}
+You can add Custom Managed Components to Zaraz, and run them as a tool.
```
Comment (kodster28) [inline content/zaraz/_index.md]:
```suggestion
You can add Custom Managed Components to Zaraz and run them as a tool.
```

**Review by kodster28 — APPROVED** [review]

**Comment by kodster28** [comment]
Any issues with this being merged in, @tomkln?

**Comment by tomkln** [comment]
Updated PR. Should be good to go. No issues from being merged on my side

File: content/zaraz/get-started/add-custom-mc.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+## Prepare a Managed Component
+
+- If you are using TypeScript, you need to have a compiled version of your Managed Component ready. Usually, this is done by running `npm run build`.
+- For vanilla JavaScript, you can skip to the deployment.
+
+## Deploy a Managed Component to Cloudflare
+
+1. Open a terminal in your managed component’s root directory.
+2. From there, run `npx managed-component-to-cloudflare-worker dist/index.js your-managed-component`, which will ask you a few questions and deploy the Managed Component to a specialised Cloudflare Worker. Change the path to your `index.js` and the name of the component to your liking.
+3. Your Managed Component should now be visible on your account as a specialized Cloudflare Worker.
```
Comment (simonabadoiu) [inline content/zaraz/get-started/add-custom-mc.md:28]: I don't see where we mention how you can really add a custom MC from the dashboard, are we saying this somewhere?

File: content/zaraz/get-started/add-custom-mc.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+## Prepare a Managed Component
+
+- If you are using TypeScript, you need to have a compiled version of your Managed Component ready. Usually, this is done by running `npm run build`.
+- For vanilla JavaScript, you can skip to the deployment.
+
+## Deploy a Managed Component to Cloudflare
+
+1. Open a terminal in your managed component’s root directory.
+2. From there, run `npx managed-component-to-cloudflare-worker dist/index.js your-managed-component`, which will ask you a few questions and deploy the Managed Component to a specialised Cloudflare Worker. Change the path to your `index.js` and the name of the component to your liking.
```
Comment (simonabadoiu) [inline content/zaraz/get-started/add-custom-mc.md]: which will deploy the Managed Component to a specialised Cloudflare Worker sounds better I think, removing the ...will ask you a few questions...

**Pull request closed without merging**